docs/sources/reference-pyroscope-architecture/about-grafana-pyroscope-architecture/index.md

```diff
@@ -59,6 +59,9 @@ The [query-scheduler]({{< relref "../components/query-scheduler" >}}) maintains
 
 The [querier]({{< relref "../components/querier" >}}) act as workers, pulling queries from the queue in the query-scheduler. The queriers connect to the ingesters to fetch all the data needed to execute a query. For more information about how the query is executed, refer to [querier]({{< relref "../components/querier.md" >}}).
```

Comment:
I realize this isn't part of your PR, i have a question about this paragraph. Should "querier" be plural or singular? It seems like querier should be plural to agree with workers and because it's plural in the subsequent sentences. 
```suggestion
The [queriers]({{< relref "../components/querier" >}}) act as workers, pulling queries from the queue in the query-scheduler. The queriers connect to the ingesters to fetch all the data needed to execute a query. For more information about how the query is executed, refer to [querier]({{< relref "../components/querier.md" >}}).
```